src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -90,3 +92,25 @@ Update "Columns" to add `links` so you can see seen any external links related t
 
 - Columns: `issue, assignee, events, title, links`
 - Sort by: `Priority`
+
+
+### Release Health
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+To monitor the health of your releases over time, you can modify the "Session Health" widget by updating the "Visualization Display" to "Area Chart":
+
+- Visualization Display: `Area Chart`
+
+Add search filters to narrow the data down to a particular release:
+
+- Search condition 1: `release:{version}`
+
+To monitor the health of your latest releases, update the Search condition to use the `latest` keyword:
```

Comment:
```suggestion
To monitor the health of your latest releases, update the search condition to use the `latest` keyword:
```

Reply:
Addressed in this commit https://github.com/getsentry/sentry-docs/pull/5083/commits/9f2e2ce97d8c0235582dbc665bc16691d5c26d36#diff-124b2f7356d01f77a1ac84d799606e8247298ade1ab46fdae30038d4afbef784R121